marray/tests/test_marray.py

```diff
@@ -310,10 +310,10 @@ def test_indexing(xp):
     # `__setitem__` can change mask
     x[1] = mxp.asarray(30, mask=False)
     assert x[1].data == 30
-    assert x[1].mask == False
+    assert not x[1].mask  # mask == array(False)
```

Comment:
was this change part of the auto-fixing of lints? I think we only need one of the two versions.

Reply:
yeah, this is https://docs.astral.sh/ruff/rules/true-false-comparison/